### PR TITLE
Renaming MCS and Mobile Client Service to MDC and Mobile Developer Co…

### PR DESCRIPTION
**Pull request by secondsun**
How to test : 

1. Have Openshift running with the ansible service broker
2. Have APB installed (you will need an older version, the latest versions have changed most of the commands)
3. apb build
4. apb push
5. Deploy Mobile Developer Console from the catalog
6. profit!

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 DOCKERHOST = docker.io
-DOCKERORG = aerogearcatalog
+DOCKERORG = sagaoftherealms
 IMAGENAME = mobile-client-service-apb
```
Comment (jhellar) [inline Makefile]: Should we also rename `IMAGENAME `?

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 DOCKERHOST = docker.io
-DOCKERORG = aerogearcatalog
+DOCKERORG = sagaoftherealms
```
Comment (jhellar) [inline Makefile]: Is this just for testing? Should not it be `aerogearcatalog`?

File: apb.yml [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
+  longDescription: "An APB deploying the AeroGear mdc-server Server with persistent storage and authentication through Openshift"
   imageUrl: "https://pbs.twimg.com/profile_images/1794440005/aerogear_icon-1_400x400.png"
-  documentationUrl: "https://docs.aerogear.org/external/apb/mcs-server.html"
+  documentationUrl: "https://docs.aerogear.org/external/apb/mdc-server.html"
   providerDisplayName: "Red Hat, Inc."
   dependencies: 
       - "docker.io/openshift/oauth-proxy:v1.1.0"
-      - "docker.io/aerogear/mobile-client-service:latest"
-  serviceName: mcs-server
+      - "docker.io/aerogearcatalog/mobile-client-service:latest"
```
Comment (jhellar) [inline apb.yml]: Should not the service (mdc server) image be in `aerogear` org as other services?

**Review by jhellar — COMMENTED** [review]
I left few comments. Also whole `roles` folder was removed. Without it it won't work, right?

**Comment by secondsun** [comment]
@jhellar role being removed was a mistake, I renamed the folders but I didn't git add them correctly ><

**Comment by secondsun** [comment]
@jhellar I think it should be aerogearcatalog and not aerogear because that is the repository mobile-core is looking at.  I can't find guidance one way or another though.

**Comment by secondsun** [comment]
@jhellar Any more thoughts?

File: roles/provision-mdc-server-apb/defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# MCS Values
+mobile_developer_console_image: "docker.io/aerogear/mobile-client-service"
```
Comment (danielpassos) [inline roles/provision-mdc-server-apb/defaults/main.yml]: Should be `mobile-developer-console` instead of `mobile-client-service`?

**Review by danielpassos — CHANGES_REQUESTED** [review]

**Comment by secondsun** [comment]
@danielpassos fixes

**Review by danielpassos — APPROVED** [review]
screenshot 2018-09-28 15 49 14

**Pull request merged**